<launch>
  <param name="device_id" value="0"/>
  <param name="calibration_path" value="$(find camera-controller)/config/camera.xml" />
  <node name="camera" pkg="camera-controller" type="camera.py" output="screen"/>
  <node name="center" pkg="camera-controller_PoC2020iot" type="image_center.py" output="screen"/>
  <node name="image_view" pkg="image_view" type="image_view" output="screen">
    <remap from="image" to="/AR/center_line" />
  </node>
</launch>